app/assets/stylesheets/application.css.scss

```diff
@@ -106,44 +112,82 @@ body h1, h2, h3, h4, h5, h6, .h1, .h2, .h3, .h4, .h5, .h6 {
       border-bottom: 4px solid #f9f9f9;
     }
   }
+
+  ul.side-menu {
+    padding-left: 0;
+  }
   ul, ul a {
     list-style-type: none;
     color: #f9f9f9;
+
   }
   ul span {
     padding-top: 8px;
-    display: block;
+
   }
   li {
-    margin-top: 20px;
-  }
-  .navbar-inverse {
-    background: none;
-    border: none;
-    position: fixed;
-    .navbar-nav {
-      .dropdown-toggle {
-        height: 120px;
-        }
-      .active>a, .active>a:hover, .active>a:focus{
-        background: transparent;
-        font-size: 1.2em;
-        }
-      .open > a, .open>a:hover, .open > a:focus{
-        background: none;
-        height: 140px;
-        transition: 0.5s;
-        z-index: 2;
-      }
+    line-height: 2em;
+    margin-top: 10px;
+    padding: 10px 0 10px 10px;
+    a {
+    display: none
+    }
+    i {
+      text-align: right;
+      padding-right: 14px;
+      float: right;
+      font-size: 22px;
     }
   }
-  .navbar-brand {
-    padding: 5px 5px;
-    height: 50px;
+}
+
+.expand{
+  left: 0;
+  width: 150px;
+  height: 100vh;
+  transition: .5s;
+  .menu-burger {
+    width: 40px;
+    height: 40px;
+    border: 2px solid #f9f9f9;
+    border-radius: 10px;
+    margin: 20px auto;
+    right: 0;
+    transition: 0.2s;
   }
 
+  ul.side-menu {
+    padding-left: 0;
+  }
+  ul, ul a {
+    list-style-type: none;
+    color: #f9f9f9;
```

Comment:
Color literals like `#f9f9f9` should only be used in variable declarations; they should be referred to via variable everywhere else.